src/Util/ColumnParser.php

```diff
@@ -16,14 +16,14 @@ class ColumnParser
      *
      * @var string
      */
-    protected $regexpParseColumn = '/^(\w*)(?::(\w*\??\[?\d*\]?))?(?::(\w*))?(?::(\w*))?/';
+    protected $regexpParseColumn = '/^(\w*)(?::(\w*\??\[?(?:\d|[0-9,])*\]?))?(?::(\w*))?(?::(\w*))?/';
```

Comment:
What you are proposing in the review below is actually to be set here.
But if we add more groups then the matches below are affected.
Both the regex at 19 (old and new) is not strict enough, it relies on user to input their fields correctly based on the documentation.

Please let me know if there is any other better way to do it.